src/Events/Calendar_Embeds/Calendar_Embeds.php

```diff
@@ -29,6 +27,24 @@ class Calendar_Embeds {
 	 */
 	const POSTTYPE = 'tec_calendar_embed';
 
+	/**
+	 * The meta key for storing the event categories.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $meta_key_categories = 'event_categories';
+
+	/**
+	 * The meta key for storing the event tags.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $meta_key_tags = 'event_tags';
```

Comment:
If they dont need to change during runtime lets gain the extra speed of a const.